### DISCUSSION TITLE
关于默认设置为中文

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#2644] [提问] [answered]
开启了默认设置为中文选项
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/c7a47841-91d6-44b4-9dca-1f0aef9313a3)
在有lang的时候就不会设置成中文了吗？
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/0fa69609-9b12-4fd4-a48c-58b806726951)
[options.txt](https://github.com/Hex-Dragon/PCL2/files/12470964/options.txt)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12470965/Log1.txt)
[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/12470966/Log2.txt)
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/12470967/Log3.txt)
[Log4.txt](https://github.com/Hex-Dragon/PCL2/files/12470968/Log4.txt)
[Log5.txt](https://github.com/Hex-Dragon/PCL2/files/12470969/Log5.txt)
我只是问一下，这可能是刻意的行为

**Comment by WForst-Breeze** [comment, marked as answer]
应该是的
